tools/sxml-srfi.scm

```diff
@@ -0,0 +1,440 @@
+      <meta charset=\"utf-8\">
+      <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">
+      <script type=\"text/x-mathjax-config\">MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\\\(','\\\\)']]}});</script>
+      <script crossorigin=\"anonymous\" integrity=\"sha384-Ra6zh6uYMmH5ydwCqqMoykyf1T/+ZcnOQfFPhDrp2kI4OIxadnhsvvA2vv9A7xYv\" src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML\" type=\"text/javascript\"></script></head>
+    <body>
+      <h1>SRFI ~a: ~a</h1>
+      <p class=\"authors\">by ~a</p>
+      <p class=\"based-on\"></p>
+      <p class=\"status\">status: <em>~a</em> (~a)</p>
+      <p class=\"keywords\">keywords: ~a</p>
+      ~a
+      <ul class=\"info\">
+        <li><a href=\"srfi-~a.html\">The SRFI Document</a></li>
+        <li><a href=\"https://srfi-email.schemers.org/srfi-~a/\">Discussion Archive</a></li>
+        <li><a href=\"https://github.com/scheme-requests-for-implementation/srfi-~a\">Git repo (on GitHub)</a></li>
+        <li><a href=\"mailto:srfi-~a@srfi.schemers.org\">srfi-~a@<span class=\"antispam\">nospam</span>srfi.schemers.org (subscribers only)</a></li>
+        <li><span class=\"firefox-column-workaround\">
+            <form method=\"POST\" action=\"https://www.simplelists.com/subscribe.php\">
+              <div class=\"title\">Subscribe to srfi-~a mailing list</div><input name=\"email\" placeholder=\"email address\" tabindex=\"1\" type=\"email\"><input name=\"name\" placeholder=\"full name\" tabindex=\"2\" type=\"text\">
+              <p><input id=\"sub-digest\" name=\"digest\" tabindex=\"3\" type=\"checkbox\" value=\"digest\"><label for=\"sub-digest\">daily digest?</label></p><input class=\"submit\" name=\"submit\" tabindex=\"4\" type=\"submit\" value=\"Subscribe to srfi-~a\"><input type=\"hidden\" name=\"action\" value=\"subscribe\"><input type=\"hidden\" name=\"list\" value=\"srfi-~a@srfi.schemers.org\"></form></span></li>
+        <li><span class=\"firefox-column-workaround\">
+            <form method=\"POST\" action=\"https://www.simplelists.com/subscribe.php\">
+              <div class=\"title\">Unsubscribe from srfi-~a mailing list</div><input name=\"email\" placeholder=\"email address\" tabindex=\"5\" type=\"email\"><input class=\"submit\" name=\"submit\" tabindex=\"6\" type=\"submit\" value=\"Unsubscribe from srfi-~a\"><input type=\"hidden\" name=\"action\" value=\"unsubscribe\"><input name=\"list\" type=\"hidden\" value=\"srfi-~a@srfi.schemers.org\"></form></span></li></ul>
+      <h2>Abstract</h2>
+      ~a
+    </body>
+</html>")
+
+(define readme-org-template "
+# SPDX-FileCopyrightText: ~a Arthur A. Gleckler
+# SPDX-License-Identifier: MIT
+* SRFI ~a: ~a
+
+** by ~a
+
+
+
+keywords: ~a
+
+This repository hosts [[https://srfi.schemers.org/srfi-~a/][SRFI ~a]]: ~a, a [[https://srfi.schemers.org/][Scheme Request for Implementation]].
+
+This SRFI is in /~a/ status.
```

Comment:
The Status section is for the editor to write.  Please don't generate that.  See `srfi-template.html`.